3.9/README.md

```diff
@@ -32,12 +32,12 @@ the nodejs itself is included just to make the npm work.
 Usage in Openshift
 ------------------
 
-For this, we will assume that you are using the supported image, available via `python:3.9` imagestream tag in Openshift.
+For this, we will assume that you are using one of the supported images, available via `python:3.9-ubi8` or `python:3.9-ubi9` imagestream tag in Openshift.
```

Comment:
Let's just use a single example here rather than a list. The users can check themselves which images is available to them.
So something like:
```suggestion
For this, we will assume that you are using one of the supported images available via imagestream tags in Openshift, eg. `python:3.9-ubi9`.
```